circle/main.go

```diff
@@ -129,10 +129,11 @@ func doEnable(flags *flag.FlagSet) error {
 func main() {
 	waitflags := flag.NewFlagSet("wait", flag.ExitOnError)
 	waitflags.Usage = func() {
-		fmt.Fprintf(os.Stderr, `usage: wait
+		fmt.Fprintf(os.Stderr, `usage: wait [refspec]
```

Comment:
@kevinburke  is `[refspec]` here something like `origin/my-feature-branch` or is it just `origin/my-feature-branch`? would it be possible to make this slightly more explicit in the helper text?